server/src/main/scala/services/ApiService.scala

```diff
@@ -73,22 +34,92 @@ abstract class ApiService(val airportConfig: AirportConfig,
 
   def actorSystem: ActorSystem
 
+  def shiftsActor: ActorRef
+
   def forecastWeekSummary(startDay: MillisSinceEpoch, terminal: Terminal): Future[Option[ForecastPeriodWithHeadlines]]
 
   def getShiftsForMonth(month: MillisSinceEpoch, terminal: Terminal): Future[ShiftAssignments]
 
   def updateShifts(shiftsToUpdate: Seq[StaffAssignment]): Unit
 
-  def getKeyCloakUsers(): Future[List[KeyCloakUser]]
+  def getShowAlertModalDialog(): Boolean
+}
 
-  def getKeyCloakGroups(): Future[List[KeyCloakGroup]]
+class ApiService(airportConfig: AirportConfig,
+                 shiftsActorRef: ActorRef,
+                 fixedPointsActor: ActorRef,
+                 staffMovementsActor: ActorRef,
```

Comment:
`fixedPointsActor` and `staffMovementsActor` are not used in this class so can be removed